darkvector.less

```diff
@@ -138,7 +138,7 @@ a,
 a:visited,
 .mw-body a.external:visited,
 .mw-body a.extiw:visited {
-    color: #D28A0D;
+    color: #af9f84;
```

Comment:
Actually, I'm ok with using a different shade of visited links. IMHO it looks like a legacy, but it's something lots of people just used to for a long time.
But I'm not sure about this specific shade, IMHO it's too pale. How about using a bit darker one?

Reply:
That was the result of some trial and error.  I could try again.  Might take some time.

Reply:
Can you please send the updated screenshot?